superset/translations/en/LC_MESSAGES/messages.po

```diff
@@ -3213,6 +3213,9 @@ msgstr ""
 #~ msgid "One ore more annotation layers failed loading."
 #~ msgstr ""
 
+#~ msgid "A valid color scheme is required"
```

Comment:
Typically we don't `i18n` as part of daily PRs, we tend to do it in batches.

Reply:
kk, removed. thanks for the heads up.